media/css/products/relay/waitlist.scss

```diff
@@ -0,0 +1,40 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/protocol/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/components/forms/form';
+@import '~@mozilla-protocol/core/protocol/css/components/forms/field';
+@import '@mozilla-protocol/core/protocol/css/components/newsletter-form';
+@import '@mozilla-protocol/core/protocol/css/includes/lib';
+
+.c-waitlist-heading {
+    text-align: center;
+
+    h1 {
+        @include font-size(38px);
+    }
+}
+
+.c-waitlist-success {
+    text-align: center;
+}
+
+.mzp-c-form {
+    i {
+        display: block;
```

Comment:
You can use [fieldnotes](https://protocol.mozilla.org/components/detail/field-notes.html) for this too.